### PR TITLE
Twilio env variables

### PR DESCRIPTION
**Pull request by Arique1104**
# Removes TWILIO_STATUS_CALLBACK_URL variable from .env.example and app.json
# (issue)

## Description

_Removes the variable TWILIO_STATUS_CALLBACK_URL from .env.example and app.json in order to _

# Checklist:

- [ ] I have manually tested my changes on desktop and mobile
- [ ] The test suite passes locally with my changes
- [ ] If my change is a UI change, I have attached a screenshot to the description section of this pull request
- [x] [My change is 300 lines of code or less](https://github.com/MoveOnOrg/Spoke/blob/main/CONTRIBUTING.md#your-first-code-contribution), or has a documented reason in the description why it’s longer
- [ ] I have made any necessary changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] My PR is labeled [WIP] if it is in progress

### CODE REVIEW

**Review by crayolakat — CHANGES_REQUESTED** [review]
Thanks! Can you also remove `TWILIO_MESSAGE_CALLBACK_URL` from `.env.example` and `app.json`? `TWILIO_MESSAGE_CALLBACK_URL` is in the same boat as `TWILIO_STATUS_CALLBACK_URL` in the fact that it should be set per organization instead of as environment variables.

References to these 2 environment variables should also be removed from `docs/HOWTO_INTEGRATE_TWILIO.md` and `docs/REFERENCE-environment_variables.md`

**Comment by Arique1104** [comment]
> Thanks! Can you also remove `TWILIO_MESSAGE_CALLBACK_URL` from `.env.example` and `app.json`? `TWILIO_MESSAGE_CALLBACK_URL` is in the same boat as `TWILIO_STATUS_CALLBACK_URL` in the fact that it should be set per organization instead of as environment variables.
> 
> References to these 2 environment variables should also be removed from `docs/HOWTO_INTEGRATE_TWILIO.md` and `docs/REFERENCE-environment_variables.md`

For additional Twilio environment variables for advanced use cases consult the Twilio Service Vendor Extension Code. 

Go look at this file: https://github.com/MoveOnOrg/Spoke/blob/main/src/extensions/service-vendors/twilio/index.js

**Comment by Arique1104** [comment]
- [ ] remove TWILIO_MESSAGE_CALLBACK_URL from .env.example and app.json? 
- [ ] TWILIO_MESSAGE_CALLBACK_URL is in the same boat as TWILIO_STATUS_CALLBACK_URL in the fact that it should be set per organization instead of as environment variables.
- [ ] Remove references to these 2 environment variables should also be removed from:
- [ ] docs/HOWTO_INTEGRATE_TWILIO.md and 
- [ ] docs/REFERENCE-environment_variables.md

**Review by crayolakat — APPROVED** [review]
Looks great!

File: app.json [hunk 1]
```diff
@@ -59,17 +59,6 @@
       "required": true
     },
 
-    "TWILIO_MESSAGE_SERVICE_SID": {
```
Comment (daradib) [inline app.json:62]: This differs from the changes to `.env.example`. Was this intended to remove `TWILIO_MESSAGE_CALLBACK_URL` instead, which was never defined in `app.json`?

**Pull request merged**